src/tools/sfdk/remoteprocess.h

```diff
@@ -124,7 +124,7 @@ private slots:
     QString m_workingDirectory;
     QSsh::SshConnectionParameters m_sshConnectionParams;
     QProcessEnvironment m_extraEnvironment;
-    bool m_runInTerminal = true;
+    bool m_runInTerminal = false;
```

Comment:
m_runInTerminal is now initialized in constructor, so initialization here has no effect.

Reply:
OTOH if I saw single member uninitialized, it would make me ponder _why_ unnecessarily, so I prefer initializing all primitive members using some safe defaults here :)